addons/event/views/event_report_templates.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="attendee_list">
+        <h1>Attendee list</h1>
+        <div class="row">
+            <div class="col-7">
+                <div t-out="event.name"/>
+            </div>
+            <div class="col-5">
+                <span t-out="event.date_begin"/>
+                <i class="fa fa-arrow-right"/>
+                <span t-out="event.date_end"/>
+            </div>
+        </div>
+        <table class="table mt-3" style="page-break-after:always;">
+            <thead>
+                <tr class="text-start">
+                    <th>Name</th>
+                    <th>Company</th>
+                    <th>Ticket type</th>
+                    <th>Phone number</th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr t-foreach="attendees" t-as="attendee">
+                    <td><t t-out="attendee.name"/></td>
+                    <td><t t-out="attendee.company_name"/></td>
+                    <td><t t-out="attendee.event_ticket_id.name"/></td>
+                    <td><t t-out="attendee.phone"/></td>
+                    <td class="text-center">
+                        <t t-if="attendee.barcode">
+                            <img t-attf-src="/report/barcode/QR/{{ attendee.barcode }}?&amp;width=87&amp;height=87&amp;quiet=0" alt="QR Code"/>
+                        </t>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+    </template>
+
+    <template id="event_registration_attendee_list">
+        <t t-call="web.html_container">
+            <t t-call="web.internal_layout">
+                <t t-foreach="docs.grouped('event_id').items()" t-as="group">
+                    <t t-call="event.attendee_list">
+                        <t t-set="event" t-value="group[0].with_context(tz=group[0].date_tz)"/>
+                        <t t-set="attendees" t-value="group[1]"/>
+                    </t>
+                </t>
+            </t>
+        </t>
+    </template>
+
+    <template id="event_event_attendee_list">
+        <t t-call="web.html_container">
+            <t t-call="web.internal_layout">
+                <t t-foreach="docs" t-as="event">
+                    <t t-call="event.attendee_list">
+                        <t t-set="event" t-value="event.with_context(tz=event.date_tz)"/>
+                        <t t-set="attendees" t-value="event.registration_ids"/>
+                    </t>
+                </t>
+            </t>
+        </t>
+    </template>
+</odoo>
```

Comment:
Maybe move these to `event_registration_report.xml`? It seems like we define the other templates in that folder without issue